tests/unit/file_export/test_transcribe_exceptions.py

```
@@ -0,0 +1,31 @@
+
+
+class TestGetAudioTextExceptionHandling:
+    def test_raises_file_export_error_on_failure(self):
+        from fastfetchbot_file_export.transcribe import get_audio_text
+
+        with patch(
+            "fastfetchbot_file_export.transcribe.OpenAI",
+            side_effect=RuntimeError("API key invalid"),
+        ):
+            with pytest.raises(FileExportError, match="Audio transcription failed"):
+                get_audio_text("/tmp/nonexistent.mp3", "fake-key")
```

Ruff (0.15.7)

[error] Probable insecure usage of temporary file or directory: "/tmp/nonexistent.mp3"

(S108)